internal/engine/interpreter/interpreter.go

```diff
@@ -212,6 +212,7 @@ type interpreterOp struct {
 	kind     wazeroir.OperationKind
 	b1, b2   byte
 	b3       bool
```

Comment:
while you're at it, could you remove `b3 bool` and reuse the `b1 byte == 1` as a bool in the next PR?

Reply:
sure!

Reply:
(probably better `b1 != 0`, I think the compiled code would be better)

Reply:
there are a few instances where this would not be possible

```
		case wazeroir.OperationITruncFromF:
			op.b1 = byte(o.InputType)
			op.b2 = byte(o.OutputType)
			op.b3 = o.NonTrapping

[...]

		case wazeroir.OperationV128ExtractLane:
			op.b1 = o.Shape
			op.b2 = o.LaneIndex
			op.b3 = o.Signed
```